Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -1144,14 +1045,100 @@ def download_file_command(client, args):
     return fileResult(filename, data=b"".join(file_chunks))
 
 
-"""Fetching"""
+@logger
+def list_watchlists_command(client, args):
+    watchlists_context = []
+    for page in client.sdk.watchlists.get_all():
+        for watchlist in page["watchlists"]:
+            watchlists_context.append(
+                {
+                    "WatchlistID": watchlist["watchlistId"],
+                    "WatchlistType": watchlist["listType"],
+                    "IncludedUsersCount": watchlist["stats"].get("includedUsersCount", 0)
+                }
+            )
+
+    if not watchlists_context:
+        CommandResults(
+            readable_output="No results found",
+            outputs_prefix="Code42.Watchlists",
+            outputs_key_field="WatchlistID",
+            outputs={"Results": []},
+            raw_response={},
+        )
+
+    readable_outputs = tableToMarkdown("Watchlists", watchlists_context)
+    return CommandResults(
+        outputs_prefix="Code42.Watchlists",
+        outputs_key_field="WatchlistID",
+        outputs=watchlists_context,
+        readable_output=readable_outputs,
+        raw_response=watchlists_context,
+    )
+
+
+@logger
+def list_watchlists_included_users(client, args):
+    watchlist = args.get("watchlist")
+    try:
+        UUID(hex=watchlist)
+        watchlist_id = watchlist
+    except ValueError:
+        watchlist_id = client.sdk.watchlists._watchlists_service.watchlist_type_id_map.get(watchlist)
+        if watchlist_id is None:
+            raise Code42InvalidWatchlistTypeError(watchlist)
+    included_users_context = []
+    for page in client.sdk.watchlists.get_all_included_users(watchlist_id):
+        for user in page["includedUsers"]:
+            included_users_context.append({"Username": user["username"], "AddedTime": user["addedTime"], "WatchlistID": watchlist_id})
+    readable_outputs = tableToMarkdown("Watchlists", included_users_context)
+    return CommandResults(
+        outputs_prefix="Code42.WatchlistUsers",
+        outputs=included_users_context,
+        readable_output=readable_outputs,
+    )
+
 
+@logger
+def add_user_to_watchlist_command(client, args):
+    username = args.get("username")
+    watchlist = args.get("watchlist")
+    user = client.get_user(username)
+    user_id = user["userUid"]
+    demisto.log(user_id)
```

Comment:
please use `demisto.info()` or `demisto.debug()` instead of `demisto.log`

Reply:
This logging was added only for testing during dev, so I removed them.